packages/edit-site/src/utils/constants.js

```diff
@@ -1,5 +1,41 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+// Navigation
+export const NAVIGATION_POST_TYPE = 'wp_navigation';
+
+// Templates.
+export const TEMPLATE_POST_TYPE = 'wp_template';
+export const TEMPLATE_PART_POST_TYPE = 'wp_template_part';
+export const TEMPLATE_CUSTOM_SOURCE = 'custom';
+
+// Patterns.
+export const PATTERN_POST_TYPE = 'wp_block';
+export const PATTERN_DEFAULT_CATEGORY = 'all-patterns';
+export const PATTERN_THEME_TYPE = 'pattern';
```

Comment:
Same as above: https://github.com/WordPress/gutenberg/blob/trunk/packages/edit-site/src/components/page-patterns/utils.js#L4

Reply:
I'm not sure, I'll take your advice on this. 

I see a lot of use of `PATTERNS` around, which `PATTERN_THEME_TYPE` replaces in this PR.

Reply:
I wonder if an object of `PATTERN_TYPES` would be clearer:

```js

export const PATTERN_TYPES = {
    theme: 'pattern',
    user: 'wp_block`
}
```
then used as `PATTERN_TYPE.theme` &  `PATTERN_TYPE.user`. User is probably more descriptive of the source than post.

Reply:
> then used as PATTERN_TYPE.theme & PATTERN_TYPE.user. User is probably more descriptive of the source than post.

If that's clearer to you folks, who have been working on this, then it sounds good to me. 😄 

By "post" I was trying to allude to the fact that it's a custom post type (inheriting the post object).